CoinDock-api/app/Console/Kernel.php

```diff
@@ -16,6 +16,7 @@ class Kernel extends ConsoleKernel
     protected function schedule(Schedule $schedule)
     {
         $schedule->command('passport:purge')->hourly();
+        $schedule->command('wallet:handle_balance')->everyFiveMinutes();
```

Comment:
can we add runInBackground() and withoutOverlapping() to these commands

Reply:
done